insights/core/__init__.py

```diff
@@ -6,12 +6,21 @@
 import re
 import shlex
 import yaml
-import xml.etree.ElementTree as ET
 from insights.contrib.ConfigParser import RawConfigParser
 
 from insights.parsers import ParseException
 from insights.core.serde import deserializer, serializer
 
+import sys
+# Since XPath expression is not supported by the ElementTree in Python 2.6,
+# import insights.contrib.ElementTree when running python is prior to 2.6 for compatibility.
+# Script insights.contrib.ElementTree is the same with xml.etree.ElementTree in Python 2.7.14
+# Otherwise, import xml.etree.ElementTree instead.
+if sys.version_info[0] == 2 and sys.version_info[1] <= 6:
+    import insights.contrib.ElementTree as ET
+else:
+    import xml.etree.ElementTree as ET
+
```

Comment:
Hey @bfahr, @kylape or @csams, Would you please take a look at this modification?

The two files ElementTree.py and ElementPath.py are copied from the latest "/usr/lib64/python2.7/xml/etree/" module.

Reply:
@bfahr @kylape @csams - If this kind of python version checking is OK, could we also carry it out on the `import` of `ConfigParser` and `RawConfigParser`?

Reply:
@xiangce would you mind to clarify the reason for including these from 2.7?  Is it because implementation changes for `ElementTree` and `ElementPath` between 2.6 and 2.7?  https://docs.python.org/2/whatsnew/2.7.html#updated-module-elementtree-1-3

Reply:
@bfahr - Sorry for missing the explanation. It's because that the `get_elements` used the XPath feature which is only available in ElementTree-1.3.  When testing on python-2.6, it reported errors like: `expected path separator ([)`, so we try to backport the ElementTree-1.3 for python-2.6 by copying them to the `contrib` folder. (we got that from https://stackoverflow.com/questions/222375/elementtree-xpath-select-element-based-on-attribute)

For `ElementPath`, it's because that the `ElementTree` depends on it.

HTH, Thanks.

Reply:
Thanks for the explanation @xiangce.